<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollAreaMap">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>650</width>
      <height>650</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaMapWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>648</width>
       <height>648</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>realtime</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelZoomed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFiltered">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="text">
     <string>Your advertisement here</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMapName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Not chosen</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxConv">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>651</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFC">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>651</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelMapNum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>ConvLayerChanged(QString)</slot>
  <slot>FCLayerChanged(QString)</slot>
 </slots>
</ui>
